configure.ac

```diff
@@ -6231,6 +6264,14 @@ fi
 
 if test "x$with_libdpdk" = "xyes"
 then
+  plugin_dpdkevents="no (DPDK version < 16.07)"
```

Comment:
There's no single symbol that would determine if  current DPDK instance supports keepalive feature, if that's what you mean. We have to check if version is >= 16.07 this way or other. It would be possible to simplify a bit this logic since DPDK header rte_version.h provides also compound macro  RTE_VERSION  with major and minor version numbers included, so I could apply it and related changes if you like it.